src/diff.js

```diff
@@ -56,15 +56,15 @@ class MonacoDiffEditor extends React.Component {
 
   editorDidMount(editor) {
     this.props.editorDidMount(editor, monaco);
-    editor.onDidUpdateDiff((event) => {
+    editor.onDidUpdateDiff(() => {
```

Comment:
According to the type(enter <https://unpkg.com/monaco-editor@0.12.0/esm/vs/editor/editor.api.d.ts> and search `onDidUpdateDiff`), the callback function has no `event` parameter.